docs/synthetics/using-the-api.md

```diff
@@ -0,0 +1,777 @@
+title: Using the API.
```

Comment:
A title shouldn't have a full stop.  It also doesn't mention Synthetics or Monitors, which may make sense from the SEO POV and simply people knowing what sort of API this is about when they see this in SERPs.

Reply:
Yes it shouldn't be there, I will remove it.